core.md

```diff
@@ -603,16 +622,36 @@ preexisting buffer.
 
 ## Prototypes
 
-Typed objects introduce complex hierarchies of prototypes. As for other objects, the
-`[[Prototype]]` of typed object instances is set to their constructor's `prototype`. The
-`prototype`s of all struct type definitions have `StructType.prototype` as their
-`[[Prototype]]`.
+Typed objects introduce several inheritance hierarchies. In addition to the
+prototype chains of struct type instances, there are those for the struct
+types themselves and for struct type arrays.
+
+In general, just like with other objects, the `[[Prototype]]` of all involved
+instances is set to their constructor's `prototype`:
+
+For struct type definitions, that means the `[[Prototype]]` is set to
+`StructType.prototype`.
+
+For instances of a struct type `PointType`, the `[[Prototype]]` is set to
+`PointType.prototype`.
+
+Finally, for instances of a struct type array `PointType.array`, the
+`[[Prototype]]` is set to `PointType.array.prototype`.
+
+### Shared Base Constructors
+
+Analogously to typed arrays, which all inherit from
+[`%TypedArray%`](https://tc39.github.io/ecma262/#sec-%typedarray%-intrinsic-object),
+Struct types and their instances inherit from shared base constructors:
 
-Analogously, a struct type array has `[type constructor].array.prototype` as its
-`[[Prototype]]`, which in turn has `StructType.array.prototype` as its `[[Prototype]]`.
-Finally, `StructType.array.prototype`'s `[[Prototype]]` is set to
-`%TypedArray%.prototype`. I.e., struct type arrays extend `%TypedArray%`, just as typed
-arrays do.
+The `[[Prototype]]` of `StructType.prototype` is `%Type%.prototype`, where
+`%Type%` is an intrinsic that's not directly exposed.
+
+The `[[Prototype]]` of `PointType.prototype` is `%Struct%.prototype`, where
+`%Struct%` is an intrinsic that's not directly exposed.
```

Comment:
We probably do eventually want some easier path to this object, right? (e.g., `StructType.prototype.prototype`) Obviously the precise path though will depend on whether we retain the "metaclass" concept.